packages/pigment-css-react/src/baseAtomics.js

```diff
@@ -0,0 +1,43 @@
+import { generateAtomics } from './generateAtomics';
+
+export const stackAtomics = generateAtomics(({ theme }) => ({
+  conditions: Object.keys(theme.breakpoints.values).reduce((acc, breakpoint) => {
+    acc[breakpoint] = `@media (min-width: ${theme.breakpoints.values[breakpoint]}${
+      theme.breakpoints.unit ?? 'px'
+    })`;
+    return acc;
+  }, {}),
+  defaultCondition: theme.breakpoints?.keys?.[0] ?? 'xs',
```

Comment:
Something to figure out. Right now, it's hardcoded if not present in theme.

Reply:
Should we figure this out before merging this PR?

Reply:
It already takes the value from theme. I made the first comment way back. It's not valid now.